### ISSUE TITLE
Firefox with Wayland support

### ISSUE THREAD

**Issue by okias** [pkulev/riru#34]
This is NOTE for me, to send you pull request about wayland firefox from https://github.com/stransky/gecko-dev . This tracks gecko-dev, but adds wayland support.

I'll try send pull in few days

**Comment by okias** [comment]
wayland support merged into upstream.

only actual change I use is:

```
-       mozconfig_annotate '' --enable-default-toolkit=cairo-gtk3
+       mozconfig_annotate '' --enable-default-toolkit=cairo-gtk3-wayland

```

**Comment by okias** [comment]
PR https://github.com/pkulev/riru/pull/44

**Comment by pkulev** [comment]
Closed by #44
